### ISSUE TITLE
vessel_target_details => HTTP 500 => Java Null Pointer Exceptions 

### ISSUE THREAD

**Issue by WSA-S** [dma-ais/AisView#2]
http://servername:8090/vessel_target_details?id=mmsi will give a HTTP500 and some Java Exceptions  on the conole.

**Comment by WSA-S** [comment]
Hi,
any workaround avaibel, to get it running with AISWEB?

Thanks

**Comment by tbsalling** [comment]
Nope; haven't had time to look into this yet. But it's still on my list.
